hybrasyl/Objects/Monster.cs

```diff
@@ -35,6 +40,14 @@ public class Monster : Creature, ICloneable
 
         private Spawn _spawn;
 
+        private Creatures.Damage _simpleDamage;
+
+        public int ActionDelay = 800;
+
```

Comment:
It occurs to me that this is a value that should be expressible in XML.

Reply:
This was a stopgap to avoid adding in extra to XML. This actually will likely be removed in the future.

Reply:
👍